### PR TITLE
Update CHANGELOG for v0.61.0

### PR DESCRIPTION
**Pull request by fpetkovski**
## Description

This commit bumps the version and updates the CHANGELOG for the v0.61.0 release.

Signed-off-by: Filip Petkovski <filip.petkovsky@gmail.com>

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: CHANGELOG.md
Comment (PhilipGough) [inline CHANGELOG.md:16]: I wonder about the need to add the previous two "bugfixes" which are docs/example updates?

Comment (fpetkovski) [reply CHANGELOG.md:16]: I had a similar thought, not sure if these are really bugs. In the case we want to have them, it could be useful to have a classification `[DOCS]`. Otherwise happy to remove them.

Comment (PhilipGough) [reply CHANGELOG.md:16]: Yeah, my thought process is that if we aren't shipping anything we should not include it but I am also ok with either model once we are consistent.

Any thoughts @simonpasquier

Comment (simonpasquier) [reply CHANGELOG.md:16]: Personally I don't include doc changes in the changelog. The categories (CHANGE, FEATURE, ...) were taken from the Prometheus project which doesn't list documentation updates.

File: CHANGELOG.md
Comment (simonpasquier) [inline CHANGELOG.md:11]:
```suggestion
* [ENHANCEMENT] Added the `filterExternalLabels` field to the remote read configuration of the `Prometheus` CRD. #5142
```

File: CHANGELOG.md
Comment (PhilipGough) [inline CHANGELOG.md:12]: We should probably remove this docs change then also!

**Review by PhilipGough — APPROVED** [review]
lgtm

File: CHANGELOG.md
Comment (simonpasquier) [inline CHANGELOG.md:4]:
```suggestion
* [CHANGE] Removed validations in the operator that are already covered at the CRD level. #5108
```

**Review by simonpasquier — APPROVED** [review]
one small nit, ready to :ship: once fixed :)

**Comment by simonpasquier** [comment]
@fpetkovski could we sneak in https://github.com/prometheus-operator/prometheus-operator/pull/5084? I think it's good to go (need to do a final review) and it would address the long-standing issue for folks that want to use AlertmanagerConfig resources without namespace enforcement.

**Comment by fpetkovski** [comment]
Sure thing, I will update the changelog once that PR is merged.

**Comment by simonpasquier** [comment]
@fpetkovski I've just merged the other PR :)

**Comment by fpetkovski** [comment]
Changelog entry should be there :)

**Review by PhilipGough — APPROVED** [review]

**Review by simonpasquier — APPROVED** [review]
:shipit:

**Pull request merged**
